components/proxy/conf/Caddyfile

```diff
@@ -223,6 +223,16 @@ https://{$GITPOD_DOMAIN} {
 		}
 	}
 
+	@local_app path /static/bin/*
+	handle @local_app {
+		import compression
+
+		reverse_proxy ide-proxy.{$KUBE_NAMESPACE}.{$KUBE_DOMAIN}:80 {
```

Comment:
It should still work in self-hosted correct? Could you check with Pudong please. As far as I know in self-hosted we don't have `ide` subdomain.